core/src/uix/compiler/alpha.cljs

```diff
@@ -352,10 +352,14 @@
             (str "/" name-part)
             demunge-name)))))
 
+(set! *format-display-name* default-format-display-name)
+
 (defn format-display-name [^string s]
-  (if *format-display-name*
-    (*format-display-name* s default-format-display-name)
-    (default-format-display-name s)))
+  (if (fn? *format-display-name*)
+    (*format-display-name* s)
+    (throw (ex-info "unexpected uix.compiler.alpha/*format-display-name* is not bound to a function"
```

Comment:
So we really need it here? It's probably user's responsibility to not assign something other than a function

Reply:
Just a sanity check, I'm ok if you want to remove it. But leave there at least assert so the stack trace is better than infamous ".call to undefined".

Reply:
I think in this case the var will be always bound because it is initialized with the default function.

Reply:
I've changed the var to be initialized with a default fn, but you are right about ".call to undefined", saw this in your tests.